index.html

```diff
@@ -8,6 +8,826 @@
+                    <div class="posted-by">
+                        <a href="#">Yaron Yitzhak</a>
+                        <span>32 minutes ago</span>
+                    </div>
+                </div>
+
+                <div class="col-1-4 col-1-3 col-4-4">
+                    <a class="img-article" href="#">
+                        <div>
+                            <div class="text-rotated">96% off</div>
+                        </div>
+                    </a>
+                    <a href="">
+                        <p>
+                            This is how rich you’d be if you bought Apple stock instead of its products
+                        </p>
+                    </a>
+                    <div class="posted-by">
+                        <a href="#">Yaron Yitzhak</a>
+                        <span>32 minutes ago</span>
+                    </div>
+                </div>
+
+                <div class="col-1-4 col-1-3 col-4-4">
+                    <a class="img-article" href="#">
+                        <div>
+                            <div class="text-rotated">96% off</div>
+                        </div>
+                    </a>
+                    <a href="">
+                        <p>
+                            This is how rich you’d be if you bought Apple stock instead of its products
+                        </p>
+                    </a>
+                    <div class="posted-by">
+                        <a href="#">Yaron Yitzhak</a>
+                        <span>32 minutes ago</span>
+                    </div>
+                </div>
+            </article>    
+        </section>
+    </main>
+
+    <footer>
+        <div class="footer-links">
+            <div class="wrapper">
+                <ul class="social-icons">
+                    <li><a href=""><i class="icon icon--facebook--light"></i></a></li>
+                    <li><a href=""><i class="icon icon--twitter--light"></i></a></li>
+                    <li><a href=""><i class="icon icon--youtube--light"></i></a></li>
+                    <li><a href=""><i class="icon icon--instagram--light"></i></a></li>
+                    <li><a href=""><i class="icon icon--pinterest--light"></i></a></li>
+                    <li><a href=""><i class="icon icon--rss--light"></i></a></li>
+                    <li><a href=""><i class="icon icon--mail--light"></i></a></li>
+                </ul>
+                <ul>
+                    <li><a href="">EVENTS</a></li>
+                    <li><a href="">ABOUT</a></li>
+                    <li><a href="">TEAM</a></li>
+                    <li><a href="">ADVERTISE</a></li>
+                    <li><a href="">JOBS</a></li>
+                    <li><a href="">CONTACT</a></li>
+                </ul>
+            </div> 
+        </div>
+        <div class="footer-legal">
+            <div class="wrapper">
+                <span>
+                    <a href="#">
+                        <img src="img/tnw.svg"/>
```

Comment:
An `img` element must have an alt attribute, except under certain conditions, correct this.